### PR TITLE
renaming example 3 module

### PR DESCRIPTION
**Pull request by ashishsingh18**
example 3

### CODE REVIEW

File: Modules/Example3/cmdapps/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+option(BUILD_CaPTkExample3CmdApp "Build command-line app for CaPTkExample3" ON)
+
+if(BUILD_CaPTkExample3CmdApp)
+  mitkFunctionCreateCommandLineApp(
+    NAME CaPTkExample3CmdApp
+    CPP_FILES CaPTkExample3CmdApp.cpp
+    PACKAGE_DEPENDS ITK Qt5|Core
```
Comment (dboun) [inline Modules/Example3/cmdapps/CMakeLists.txt]: I don't think there is any qt used in the cmd app.

File: Modules/Example3/cmdapps/CaPTkExample3CmdApp.cpp [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+  // How should arguments be prefixed
+  parser.setArgumentPrefix("--", "-");
+
+  /*---- Add arguments. Unless specified otherwise, each argument is optional.
+            See mitkCommandLineParser::addArgument() for more information. ----*/
+
+  parser.addArgument(
+    "input",
+    "i",
+    mitkCommandLineParser::String,
```
Comment (dboun) [inline Modules/Example3/cmdapps/CaPTkExample3CmdApp.cpp]: I think there is _mitkCommandLineParser::File_ which is technically more correct.

**Review by dboun — COMMENTED** [review]
I though the plan was for ex3 to be transferring the "important" functionality from the plugin to the module, but still showcasing everything through the plugin. 

And then ex4 building a cmd app for the module.

File: Modules/Example3/cmdapps/CMakeLists.txt (hunk 1 above)
Comment (ashishsingh18) [reply Modules/Example3/cmdapps/CMakeLists.txt]: yes, you are right I will create example4 that will have cmd app.

**Comment by ashishsingh18** [comment]
> 
> 
> I though the plan was for ex3 to be transferring the "important" functionality from the plugin to the module, but still showcasing everything through the plugin.
> 
> And then ex4 building a cmd app for the module.

yes, you are right I will create example4 that will have cmd app.

File: Modules/Example3/cmdapps/CaPTkExample3CmdApp.cpp (hunk 2 above)
Comment (ashishsingh18) [reply Modules/Example3/cmdapps/CaPTkExample3CmdApp.cpp]: another PR for example4 coming.

**Review by AlexanderGetka-cbica — COMMENTED** [review]
Makes sense, approving

**Review by AlexanderGetka-cbica — APPROVED** [review]

**Review by AlexanderGetka-cbica — APPROVED** [review]

**Comment by AlexanderGetka-cbica** [comment]
@ashishsingh18 Approval seems to work now. What changed?

**Pull request merged**